Index: src/test/resources/testngRegistration.xml
===================================================================
diff --git a/src/test/resources/testngRegistration.xml b/src/test/resources/testngRegistration.xml
deleted file mode 100644
--- a/src/test/resources/testngRegistration.xml	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
+++ /dev/null	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
-<suite name="testRegistration" verbose="1">
-    <test name="RegistrationTest">
-        <classes>
-            <class name="e2e.tests.registration.RegistrationAndAuthorisationAdminTest">
-            </class>
-            <class name="e2e.tests.adminPanel.AdminDeleteUserTest">
-            </class>
-        </classes>
-    </test>
-</suite>
\ No newline at end of file
Index: src/test/java/e2e/tests/profile/EditUserDataProfileTest2.java
===================================================================
diff --git a/src/test/java/e2e/tests/profile/EditUserDataProfileTest2.java b/src/test/java/e2e/tests/profile/EditUserDataProfileTest2.java
deleted file mode 100644
--- a/src/test/java/e2e/tests/profile/EditUserDataProfileTest2.java	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
+++ /dev/null	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
@@ -1,100 +0,0 @@
-package e2e.tests.profile;
-
-import com.github.javafaker.Faker;
-import e2e.TestBase;
-import e2e.enums.GenderInfo;
-import e2e.enums.SideBarInfo;
-import e2e.pages.Header;
-import e2e.pages.homeBlog.HomeBlogPage;
-import e2e.pages.login.LoginPage;
-import e2e.pages.profile.EditPasswordForm;
-import e2e.pages.profile.EditUserForm;
-import io.qameta.allure.*;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
-import java.time.LocalDate;
-import java.time.format.DateTimeFormatter;
-
-public class EditUserDataProfileTest2 extends TestBase {
-
-    Faker faker = new Faker();
-    LoginPage loginPage;
-    Header header;
-    HomeBlogPage homeBlogPage;
-    EditUserForm editUserForm;
-    EditPasswordForm editPasswordForm;
-
-    private void checkEditUserData(EditUserForm page, String name, String surname, String date, String phone) {
-        String actualName = page.getName();
-        String actualSurname = page.getSurname();
-        String actualDate = page.getDate();
-        String actualPhone = page.getPhone();
-        Assert.assertEquals(actualName, name, actualName + " is not equal " + name);
-        Assert.assertEquals(actualSurname, surname, actualSurname + " is not equal " + surname);
-        Assert.assertEquals(actualDate, date, actualDate + " is not equal " + date);
-        Assert.assertEquals(actualPhone, phone, actualPhone + " is not equal " + phone);
-    }
-
-    @Epic(value = "User can edit data to the profile")
-    @Feature(value = "User edited data to the profile")
-    @Description(value = "User can edit data")
-    @Severity(SeverityLevel.CRITICAL)
-    @AllureId("")
-    @Test(description = "CHATTY-30")
-    public void userCanEditProfile() {
-
-        String email = "tatar@abv.bg";
-        String password = "Manowar333246";
-
-        String editName = "Georgiy";
-        String editSurname = "Manolov";
-        String editFormattedDate = "03-01-1985";
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
-        LocalDate birthDate = LocalDate.parse(editFormattedDate, formatter);
-
-
-        String editPhone = "+4915731078";
-        String editImageAvatar = "C:\\Users\\ACER\\IdeaProjects\\Chatty\\avatar";
-
-        String oldPassword = "Manowar333246";
-        String newPassword = "Manowar33246";
-        String confirmNewPassword = "Manowar33246";
-
-        loginPage = new LoginPage(app.driver);
-        loginPage.login(email, password);
-
-        homeBlogPage = new HomeBlogPage(app.driver);
-        homeBlogPage.waitForLoading();
-
-        header = new Header(app.driver);
-        header.tabDropdownMenu(SideBarInfo.USERPROFILE);
-
-        editUserForm = new EditUserForm(app.driver);
-        editUserForm.waitForLoading();
-        editUserForm.waitForLoading();
-        editUserForm.waitForLoading();
-        editUserForm.clickEditUserForm();
-        editUserForm.waitForLoading();
-
-        editUserForm.setEditProfileForm(editName, editSurname, GenderInfo.MALE, birthDate.format(DateTimeFormatter.ofPattern("yyyy-MM-dd")), editPhone);
-        editUserForm.waitForLoading();
-        editUserForm.saveButtonClick();
-        editUserForm.waitForLoading();
-        checkEditUserData(editUserForm, editName, editSurname, editFormattedDate, editPhone);
-
-        editPasswordForm = new EditPasswordForm(app.driver);
-        editPasswordForm.fillChangePasswordForm(oldPassword, newPassword, confirmNewPassword);
-        editPasswordForm.clickSaveChangePasswordButton();
-
-        header = new Header(app.driver);
-        header.clickHome();
-        header.tabDropdownMenu(SideBarInfo.LOGIN);
-
-        loginPage = new LoginPage(app.driver);
-        loginPage.login(email, confirmNewPassword);
-
-        homeBlogPage = new HomeBlogPage(app.driver);
-        homeBlogPage.waitForLoading();
-    }
-}
Index: src/test/java/integration/pages/user/UserLoginApi.java
===================================================================
diff --git a/src/test/java/integration/pages/user/UserLoginApi.java b/src/test/java/integration/pages/user/UserLoginApi.java
deleted file mode 100644
--- a/src/test/java/integration/pages/user/UserLoginApi.java	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
+++ /dev/null	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
@@ -1,43 +0,0 @@
-package integration.pages.user;
-
-import io.restassured.RestAssured;
-import io.restassured.response.Response;
-
-public class UserLoginApi {
-
-    public static void main(String[] args) {
-        String baseUrl = "http://chatty.telran-edu.de:8989";
-        String endpoint = "/api/auth/login";
-
-
-        String email = "tatar@abv.bg";
-        String password = "Manowar33246";
-
-
-        Response response = RestAssured.given()
-                .baseUri(baseUrl)
-                .basePath(endpoint)
-                .header("accept", "*/*")
-                .header("Content-Type", "application/json")
-                .body("{ \"email\": \"" + email + "\", \"password\": \"" + password + "\" }")
-                .post();
-
-
-        int statusCode = response.getStatusCode();
-        System.out.println("Response Code: " + statusCode);
-
-
-        String responseBody = response.getBody().asString();
-        System.out.println("Response Body: " + responseBody);
-
-
-        if (statusCode == 200) {
-            String accessToken = response.jsonPath().getString("accessToken");
-            String refreshToken = response.jsonPath().getString("refreshToken");
-            long expiration = response.jsonPath().getLong("expiration");
-            System.out.println("Access Token: " + accessToken);
-            System.out.println("Refresh Token: " + refreshToken);
-            System.out.println("Expiration: " + expiration);
-        }
-    }
-}
Index: src/test/resources/testngE2eSmoke2.xml
===================================================================
diff --git a/src/test/resources/testngE2eSmoke2.xml b/src/test/resources/testngE2eSmoke2.xml
deleted file mode 100644
--- a/src/test/resources/testngE2eSmoke2.xml	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
+++ /dev/null	(revision 453aa9dcc30cd779379ce3deebaefef81a889044)
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
-<suite name="testE2eSmoke2" verbose="1">
-    <test name="SmokeTestE2e2">
-        <classes>
-            <class name="e2e.tests.registration.RegistrationAndAuthorisationUserTest">
-            </class>
-            <class name="e2e.tests.profile.AddUserDataProfileTest">
-            </class>
-            <class name="e2e.tests.profile.EditUserDataProfileTest2">
-            </class>
-            <class name="e2e.tests.contactUs.ContactUsPositiveTest">
-            </class>
-            <class name="e2e.tests.about.AboutPageTest">
-            </class>
-            <class name="e2e.tests.post.UserCanCreateANewPost">
-            </class>
-            <class name="e2e.tests.post.UserCanEditPostTest">
-            </class>
-            <class name="e2e.tests.post.UserCanDeletePostTest">
-            </class>
-            <class name="e2e.tests.adminPanel.AdminDeleteUserTest">
-            </class>
-        </classes>
-    </test>
-</suite>
\ No newline at end of file
